lib/tasks/scheme_twelve.rake

```diff
@@ -0,0 +1,27 @@
+require Rails.root.join('db','seeds', 'schemas', 'add_agfs_fee_scheme_12')
+
+namespace :db do
+  namespace :scheme_twelve do
+    desc 'Display status of db structures and records for AGFS fee scheme 12'
+    task :status => :environment do
+      adder = Seeds::Schemas::AddAgfsFeeScheme12.new(pretend: false)
+      puts adder.status
```

Comment:
Debug code? (can this be removed?)

Reply:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3432#discussion_r463064520)